ext/session/tests/session_set_save_handler_class_017.phpt

```diff
@@ -55,7 +55,7 @@ class MySession2 extends SessionHandler {
 	}
 
 	public function create_sid() {
-		return 'my_sid';
+		return 'session_set_save_handler_class_017';
```

Comment:
maybe use `basename(__FILE__)` or similar so the SID cannot collide when tests get copied again?

Reply:
I like the idea.
I used `pathinfo(__FILE__)['filename']` because it felt weird to have session files with a `.phpt` extension.